etomo/tests/fourier_reconstruction.py

```diff
@@ -0,0 +1,58 @@
+"""
+Example of a single image reconstruction
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+import pysap
+from pysap.data import get_sample_data
+from modopt.math.metrics import ssim
+from modopt.opt.linear import Identity
+from modopt.opt.proximity import SparseThreshold
+
+from etomo.operators.utils import generate_locations_etomo_2D
+from etomo.operators import NFFT, WaveletPywt, HOTV
+from etomo.reconstructors.forwardradon import RadonReconstructor
+from etomo.operators.fourier.utils import estimate_density_compensation
+
+
+# Loading input data
+image = get_sample_data('2d-mri')
+img_size = image.shape[0]
+
+# Create radon operator and simulate data
+theta = np.arange(0., 180., 3.)
+kspace_loc = generate_locations_etomo_2D(img_size, theta)
+fourier_op = NFFT(samples=kspace_loc, shape=image.shape, normalized=True)
+data = fourier_op.op(image.data)
+
+TV = HOTV(img_shape=image.shape, order=1)
+wavelet = WaveletPywt(wavelet_name='sym8', nb_scale=3)
+linear_op = TV
+
+regularizer_op = SparseThreshold(linear=Identity(), weights=3e-6)
+reconstructor = RadonReconstructor(
```

Comment:
Same comment. If you think it's better to keep a single class for image recon, I'm fine with that but we should probably change the className to ETomoReconstructor. WDYT??